util/log/file.go

```diff
@@ -165,37 +155,33 @@ func parseLogFilename(filename string) (FileDetails, error) {
 	}, nil
 }
 
-var onceLogDirs sync.Once
-
 // create creates a new log file and returns the file and its filename, which
-// contains severity ("INFO", "FATAL", etc.) and t.  If the file is created
+// contains severity ("INFO", "FATAL", etc.) and t. If the file is created
 // successfully, create also attempts to update the symlink for that tag, ignoring
 // errors.
 func create(severity Severity, t time.Time) (f *os.File, filename string, err error) {
-	onceLogDirs.Do(createLogDirs)
-	if len(logDirs) == 0 {
-		return nil, "", errors.New("log: no log dirs")
+	if len(*logDir) == 0 {
+		return nil, "", errors.New("log: log directory empty")
```

Comment:
s/empty/unset/

current wording implies that the log directory doesn't contain files